src/Service.Host/client/src/index.js

```diff
@@ -22,7 +24,9 @@ const render = Component => {
             <ThemeProvider theme={linnTheme}>
                 <SnackbarProvider dense maxSnack={5}>
                     <AppContainer>
-                        <Component store={store} />
+                        <LocalizationProvider dateAdapter={AdapterDateMoment} locale="de-DE">
```

Comment:
local de-DE? what does that do?

Reply:
Ahh I meant to post about this, the date was in American format in datepickers and I couldn't get it to accept en-GB sort of thing, german was at least the right way round just with .s ie 16.12.21

Reply:
I'll post in it dev actually, definitely only a temporary thing